lib/protoparser/opentelemetry/stream/streamparser.go

```diff
@@ -207,9 +220,15 @@ func (wr *writeContext) appendSampleWithExtraLabel(metricName, labelName, labelV
 
 // appendAttributesToPromLabels appends attributes to dst and returns the result.
 func appendAttributesToPromLabels(dst []prompbmarshal.Label, attributes []*pb.KeyValue) []prompbmarshal.Label {
+	var sanitizeFn func(string) string
```

Comment:
I think, it's better to specify default realization. It reduces complexity and better from reader PoV. 

```go
sanitizeFn := func(labelName string) string { return labelName }
```

 And override it if needed:
```go
if *sanitizeMetrics {
		sanitizeFn = promrelabel.SanitizeLabelName
}
```